File: json-to-messages/ruby/features/hooks/hooks_steps.rb
Before() do
  # no-op
end

Before('@beforeHook') do
  # no-op
end

Before('@failBeforeHook') do
  raise 'Woops !'
end

After('@afterHook') do
  # no-op
end

Given('a passed step') do
  # no-op
end

File: json-to-messages/javascript-testdata/features/hooks/hooks_steps.ts
import { Given, Before, After } from '@cucumber/cucumber'

Before(() => {
  //no-op
})

Before('@beforeHook', () => {
  //no-op
})

Before('@failBeforeHook', () => {
  throw new Error('Woops !')
})

After('@beforeHook', () => {
  //no-op
})

Given('a passed step', () => {
  //no-op
})
